.stylelintrc.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
       "declaration-empty-line-before": null,
       "declaration-colon-newline-after": null,
       "declaration-no-important": true,
-      "max-nesting-depth": 1,
+      "max-nesting-depth": 3,
```

Comment:
nesting depth is 2, max compound selector should go back to 1. We want to enforce these rules and not change them to our convenience.